### PR TITLE
Items i #124 

### LINKED ISSUE #124: User Story 14, Items Index Page
```
As any kind of user on the system
I can visit the items catalog ("/items")
I see all items in the system except disabled items
Each item will display the following information:
- the name of the item
- a small thumbnail image for the item
- the merchant name who sells the item
- how many of the item the merchant has in stock
- the merchant's current price for the item

The item name is a link to that item's show page
The item thumbnail is a link to that item's show page
```

### Checklist:

- [ ] base code tests are written
- [ ] base code is written to pass tests
- [ ] edge case ("sad path") tests written (if required)
- [ ] edge case code written (if required)
- [ ] code is reviewed
- [ ] branch is merged
- [ ] all tests still pass

child of #125

depends on #139

### PR DESCRIPTION
**Pull request by stoic-plus**
## Items Index Page - Closes #124 

* Showing item information
* linking to item show page from item name and thumbnail
  *  Current thumbnail is not actually an image

### CODE REVIEW

File: app/models/Item.rb [hunk 1]
```diff
@@ -1,6 +1,7 @@
 class Item < ApplicationRecord
-  validates_presence_of :name, :description, :thumbnail, :price, :inventory, :enabled
+  validates_presence_of :name, :description, :thumbnail, :price, :inventory
```
Comment (nicovigil1) [inline app/models/Item.rb]: lol nice catch

File: app/views/items/index.html.erb [hunk 2]
```diff
@@ -1,9 +1,11 @@
 <% @items.each do |i| %>
-  <div class='item' id="item-<%= i.id %>">
-    <%= i.name %>
-    <%= i.thumbnail %>
-    <%= i.user.name %>
-    <%= i.price %>
-    <%= i.inventory %>
-  </div>
+  <% if i.enabled %>
```
Comment (nicovigil1) [inline app/views/items/index.html.erb]: noice
